specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1940 @@
+        "modelId": {
+          "description": "Model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Status of the model.",
+          "enum": [
+            "creating",
+            "ready",
+            "invalid"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ModelStatus",
+            "modelAsString": false
+          },
+          "x-nullable": false
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Date and time (UTC) when the model was created.",
+          "type": "string",
+          "x-nullable": false
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Date and time (UTC) when the status was last updated.",
+          "type": "string",
+          "x-nullable": false
+        },
+        "modelName": {
+          "type": "string",
+          "description": "Optional user defined model name."
+        }
+      }
+    },
+    "Models": {
+      "description": "Response to the list custom models operation.",
+      "type": "object",
+      "properties": {
+        "summary": {
+          "description": "Summary of all trained custom models.",
+          "type": "object",
+          "required": [
+            "count",
+            "limit",
+            "lastUpdatedDateTime"
+          ],
+          "properties": {
+            "count": {
+              "description": "Current count of trained custom models.",
+              "type": "integer",
+              "x-nullable": false
+            },
+            "limit": {
+              "description": "Max number of models that can be trained for this account.",
+              "type": "integer",
+              "x-nullable": false
+            },
+            "lastUpdatedDateTime": {
+              "format": "date-time",
+              "description": "Date and time (UTC) when the summary was last updated.",
+              "type": "string",
+              "x-nullable": false
+            }
+          }
+        },
+        "modelList": {
+          "description": "Collection of trained custom models.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelInfo"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next page of custom models.",
+          "type": "string"
+        }
+      }
+    },
+    "Model": {
+      "description": "Response to the get custom model operation.",
+      "type": "object",
+      "required": [
+        "modelInfo"
+      ],
+      "properties": {
+        "modelInfo": {
+          "$ref": "#/definitions/ModelInfo"
+        },
+        "keys": {
+          "$ref": "#/definitions/KeysResult"
+        },
+        "trainResult": {
+          "$ref": "#/definitions/TrainResult"
+        },
+        "composedTrainedResults": {
```

Comment:
Would be good to have a description saying what is mapped to what in this dictionary.  I'm assuming modelId to TrainResult?  Would the friendly name of the model be exposed here too somewhere?

Reply:
For consistency, should this be composedTrainResults instead of composedTrain**ed**Results?